OrbitGl/TimeGraph.cpp

```diff
@@ -214,6 +215,26 @@ void TimeGraph::PanTime(int a_InitialX, int a_CurrentX, int a_Width,
   NeedsUpdate();
 }
 
+//-----------------------------------------------------------------------------
+void TimeGraph::VerticalAlign(const TextBox* a_TextBox) {
+  if (!a_TextBox) return;
+  if (a_TextBox->GetPos() == Vec2::Zero())  // Have to update the information of
+                                            // this TextBox before align
+    UpdatePrimitives();
```

Comment:
We should use NeedsUpdate() instead.

Reply:
Are you sure?
In theory, a_TextBox position rarely could be exactly (0.0,0.0), but I was having that issue (a_TextBox hadn't have setted the position) and this makes the function works strangely.  So, Updating primitives solved that (it gaves the TextBox a position). 

I think that the problem is when the text box isn't in the screen, and after making an horizontal alignment, we haven't updated the TextBox position. This line solved the problem, because update before the vertical alignment. But, as I'm changing things, let me see if I can avoid writing this update here.

Reply:
Still with the same issue... Maybe it is a bug from somewhere else or there is a better way to make this update needed in VerticallyScrollIntoView.

Reply:
Did you see the changes on #791? If you call NeedsUpdate() it will now set m_NeedsRedraw to true, which triggers a redraw, which also calls UpdatePrimitives. So, if UpdatePrimitives() works but NeedsUpdate() doesn't, then maybe you need to rebase on the changes in #791.

Reply:
You are right. I was making the NeedsUpdate() only at the last.

Reply:
Done!